# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1475089982 14400
#      Wed Sep 28 15:13:02 2016 -0400
# Node ID ead11c8bd7019ee562daa1a064556d1709902a8f
# Parent  73a85c6cacd18938dbf01d01114e2e6c0dcb6d5c
maint: Bump version on default.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,24 +14,24 @@ dnl Copyright (C) 1993-2016 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.63])
-AC_INIT([GNU Octave], [4.2.0-rc2], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [4.3.0+], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=4
-OCTAVE_MINOR_VERSION=2
-OCTAVE_PATCH_VERSION=0-rc2
+OCTAVE_MINOR_VERSION=3
+OCTAVE_PATCH_VERSION=0+
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2016 John W. Eaton and others."
 
 OCTAVE_RELEASE_DATE="2016-09-20"
 
